sql/plsql/PlSqlLexer.g4

```diff
@@ -1532,6 +1532,7 @@ SERIALLY_REUSABLE:            'SERIALLY_REUSABLE';
 SERIAL:                       'SERIAL';
 SERVERERROR:                  'SERVERERROR';
 SERVICE_NAME_CONVERT:         'SERVICE_NAME_CONVERT';
+SERVICE:                      'SERVICE';
```

Comment:
If I understand correctly, it's not truly keyword. That's why it should be added either to [non_reserved_keywords_in_12c](https://github.com/antlr/grammars-v4/blob/master/sql/plsql/PlSqlParser.g4#L4981) or to [non_reserved_keywords_pre12c](https://github.com/antlr/grammars-v4/blob/master/sql/plsql/PlSqlParser.g4#L5295) to make it possible to use `SERVICE` word as identifier.